business_requirement_versioning/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Business Requirement Versioning',
+    'category': 'Business Requirements Management',
+    'summary': 'update the busuness requirement version',
+    'version': '10.0.1.0.0',
+    'website': 'https://www.elico-corp.com/',
```

Comment:
@lonelysun IMHO change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)